index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gerg Capston project</title>
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <div class="page-header padding-x">
+        <ul>
+            <li><a href=""> <img src="imgs/icons/github.png" alt=""></a></li>
+            <li><a href=""> Email</a></li>
+            <li><a href=""> twiter </a></li>
+            <li><a href=""> My Page </a></li>
+            <li><a href=""> Logout </a></li>
+        </ul>
+    </div>
+    <nav class="nav_bar padding-x">
+        <a href="index.html"><img class="my-logo" src="./imgs/icons/logo.png" alt="My Log"></a>        
+        <img class="humburger" src="./imgs/icons/humburger.jpg" alt="Humburger Icon">
+        <div class="close-btn">
+            <p>x</p>
+        </div>
+        <ul>
+            <li><a class="about" href="index.html" >Home</a></li>
+            <li><a class="" href="about.html" >About</a></li>
+            <li><a href="" >Programs</a></li>
+            <li><a href="" >Join</a></li>
+            <li><a href="" >Sponser</a></li>
+            <li><a href="" >News</a></li>
+            <li  class="cc" ><ahref="" >CC campign</a></li>
+        </ul>
+    </nav>
+    <section class="home-section padding-x">
+        <h3>"Hello! Sharing World"</h3>
+        <h1>CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+        <div>
+            <p>
+                A joy full celebration beleving in the value of opennes and Sharing
+                , Creating a posetive change with people from over 80 countries is taking
+                place in Ocutober. In Korea
+            </p>
+        </div>
+        <p class="lg-text">2015.10.15(THU) ~ 16(FRI)</p>
+        <p>@ National Museum of Korea, Art Center Nabi and More</p>
+    </section>
+    <section class="main-program padding-x bg-black">
```

Comment:
The cards in the mobile version of the main program section are causing a horizontal scroll in your webpage. Please fix this so it matches the behance design
<img width="172" alt="image" src="https://user-images.githubusercontent.com/37116322/234414756-a27b5013-45e1-4256-87be-20ae73d760a8.png">

Reply:
## 🎁🎁🎁 Thanks a lot, I will try my best to correct it. 🎀🎀🎀